src/components/ImagesCarouselItem.js

```diff
@@ -13,10 +14,14 @@ import { Image } from './Image';
 export const ImagesCarouselItem = memo(
   ({ navigation, source, message, containerStyle, aspectRatio, refreshInterval }) => {
     const { routeName, params } = source;
+    //for testing
+    console.warn('capitation:', source.captionText);
 
     if (routeName && params) {
       return (
         <TouchableOpacity
+          accessibilityLabel={`(${source.captionText}) ${consts.a11yLabel.imageCarousel}`}
```

Comment:
you can see in your `console.warn` output, that `captionText` is not necessarily defined. so you should have some kind of sensible fallback here

Reply:
solved f90ab7f07ff605af93121f7569894af606fd71e3

Reply:
can you explain, what you think the issue is, and how your change solves it?

Reply:
<img width="571" alt="Screenshot 2021-07-08 at 14 14 18" src="https://user-images.githubusercontent.com/43641321/124919853-fcbb1500-dff6-11eb-97f8-0d0f88d39968.png">
for me this explains the issue and how adding ?. solves it

Reply:
done in d906981

Reply:
i don't think the new change does, what you think it does.

Reply:
right! what I need here is a "&&" symbol! done in ef58ac7be4e00799d2a955770af47c10df993eed